### DISCUSSION TITLE
Table 22, Panel 2

### DISCUSSION THREAD

**Discussion by erjank** [cssi-pi-community/2022-meeting#43] [Panel 2 roundtable (Monday afternoon)]
Data Integrity and Provenance discussion

**Comment by ludaesch** [comment]
A comment on a question to the panel: Are there publishers that require provenance as part of the submission and peer-review process? 

There are journals that require submission of data and code artifacts. In addition to the scientific reviewers there are additional "verifiers" who check that the artifacts mentioned in the paper (data and scripts) are available and produce the artifacts reported in the paper. For example, some AEA journals have this requirement (see Lars Vilhuber's work and projects).

**Comment by ludaesch** [comment]
Another comment on a question to the panel: Are there languages for provenance and repositories for provenance that one can use? A questions not asked: Are there interoperability standards for provenance?

On the latter: yes. In particular, W3C PROV. However, one needs extensions to PROV to account for the many additional provenance-related relations that one might want to represent and query. For example, ProvONE,  was developed as an extension to W3C-PROV to add prospective provenance (workflow graphs) and link it with retrospective provenance information.

**Reply by erjank** [reply]
Examples from social science journals (R, Stada scripts) that have artefact review - would be small tweak to add provenance too.

**Reply by erjank** [reply]
CoMSYS does model review, of whether it runs, whether it's documented, but not evaluating the model's quality. Does the model do what it says it's going to do?

**Comment by erjank** [comment]
Discussion of archival resources like zenodo, osf.io, european repository for data already exist and maybe aren't widely known?

**Comment by erjank** [comment]
Add service requirement to research software professionals to review data/code.

**Comment by erjank** [comment]
Discussion of how practicing data provenance would minimize fabrication or falsification of data in things like scientific images
